src/plugins/condition/components/ConditionCollection.vue

```diff
@@ -207,42 +210,47 @@ export default {
             }
         },
         addCondition(event, isDefault) {
-            let conditionDomainObject = this.getConditionDomainObject(!!isDefault);
+            let conditionDomainObject = this.createConditionDomainObject(!!isDefault);
             //persist the condition domain object so that we can do an openmct.objects.get on it and only persist the identifier in the conditionCollection of conditionSet
-            this.openmct.objects.mutate(conditionDomainObject, 'created', new Date());
             this.conditionCollection.unshift(conditionDomainObject.identifier);
             this.persist();
         },
         updateCurrentCondition(identifier) {
             this.currentConditionIdentifier = identifier;
+            // console.log('conditionCollection this.conditionIdentifier', this.currentConditionIdentifier);
         },
-        getConditionDomainObject(isDefault) {
+        createConditionDomainObject(isDefault) {
             let conditionObj = {
                 isDefault: isDefault,
+                name: isDefault ? 'Default' : 'Unnamed Condition',
                 identifier: {
                     namespace: this.domainObject.identifier.namespace,
                     key: uuid()
                 },
-                definition: {
+                configuration: {
                     name: isDefault ? 'Default' : 'Unnamed Condition',
                     output: 'false',
                     trigger: 'any',
                     criteria: isDefault ? [] : [{
+                        telemetry: '',
                         operation: '',
                         input: '',
-                        metaDataKey: '',
-                        key: ''
+                        metadata: '',
+                        identifier: {
+                            namespace: '',
+                            key: this.telemetryObjs.length ? this.openmct.objects.makeKeyString(this.telemetryObjs[0].identifier) : null
```

Comment:
The key here should always be empty as we don't want a new criterion to automatically have some telemetry selected.

Reply:
done